source/common/http/conn_manager_config.h

```diff
@@ -20,6 +20,7 @@ namespace Http {
  * All stats for the connection manager. @see stats_macros.h
  */
 #define ALL_HTTP_CONN_MAN_STATS(COUNTER, GAUGE, HISTOGRAM)                                         \
+  COUNTER(dec_buffer_overflow_total)                                                               \
```

Comment:
"decoder_buffer_overflow_total" ? Spelled out might be easier to understand as we don't do this anywhere else but up to you. Same comment though about overflow stat comments.

Reply:
Sure.